src/BinSkim.Driver/MultithreadedAnalyzeCommand.cs

```diff
@@ -129,6 +129,12 @@ internal static string ReturnCommonPathRootFromTargetSpecifiersIfOneExists(IEnum
 
         public override int Run(AnalyzeOptions analyzeOptions)
         {
+            // Check whether target file is specified for BinSkim Analyze. If not, exit with code 1 (failure).
+            if (analyzeOptions.TargetFileSpecifiers == null || !analyzeOptions.TargetFileSpecifiers.Any())
+            {
+                throw new ArgumentNullException("Please specify a targe file after BinSkim analyze", (Exception)null);
```

Comment:
>targe [](http://example.com/codeflow?start=66&length=5)

1. spelling targe -> target
2. I think this does not need to be a file, please check what can be in `TargetFileSpecifiers`
3. If sentence, end with `.`
4. I think, change `after` to `for`